Packs/Campaign/ReleaseNotes/2_0_7.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+##### FindEmailCampaign
+- Split the email summary section to summary and snippets sections and changed the output to be more readable.
```

Comment:
where was this change made? in the human readable section?

Reply:
the new field is the snippets section from the hr.